### PR TITLE
legacy stylecombo list behaviour

### LINKED ISSUE #83: Revert list behaviour to previous
Don't use the presets & detection anymore.

### PR DESCRIPTION
**Pull request by caffodian**
fixes #83

### CODE REVIEW

**Comment by caffodian** [comment]
Going to give up on fixing Travis for now.  it runs locally and passes tests, but the change to ubuntu 14.04 by default has broken things in a way, and even trying to manually make it use 12.04 again and all the old versions of things, is failing pretty hard

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 language: node_js
+dist: precise
+cache: yarn
+sudo: required
+addons:
+  firefox: "31.0"
+  phantomjs: "1.9.8"
```
Comment (petereilly-rldatix) [inline .travis.yml:7]: Do these changes fix something, or is travis still broken with them? I wonder if they'll just end up muddying the waters for getting travis fixed if they're not doing something....

Comment (caffodian) [reply .travis.yml:7]: It is still broken, but less broken (it runs a few tests before timing out, as opposed to just instantly timing out)

Comment (caffodian) [reply .travis.yml:7]: could just remove

Comment (petereilly-rldatix) [reply .travis.yml:7]: Up to you, I suppose—if these changes move things farther along, they're fine by me.

**Review by petereilly-rldatix — APPROVED** [review]

**Pull request merged**